examples/ws2812/digispark.go

```diff
@@ -0,0 +1,10 @@
+// +build digispark
+
+package main
+
+import "machine"
+
+// This is the pin assignment for the Digispark only.
+// Replace neo in the code below to match the pin
+// that you are using if different.
+var neo machine.Pin = 0
```

Comment:
This could be `const` instead of `var`. In practice it probably won't matter with the interp optimization pass, but `const` is a much stronger signal to the compiler.